src/benchmarks/micro/libraries/System.Text.Json/Utf8JsonReader/Perf.Get.cs

```diff
@@ -0,0 +1,148 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using BenchmarkDotNet.Attributes;
+using MicroBenchmarks;
+
+namespace System.Text.Json.Tests
+{
+    [BenchmarkCategory(Categories.Libraries, Categories.JSON)]
+    public class Perf_Get
+    {
+        private static readonly byte[] _jsonSByteBytes = Encoding.UTF8.GetBytes(sbyte.MaxValue.ToString());
+        private static readonly byte[] _jsonInt16Bytes = Encoding.UTF8.GetBytes(short.MaxValue.ToString());
+        private static readonly byte[] _jsonInt32Bytes = Encoding.UTF8.GetBytes(int.MaxValue.ToString());
+        private static readonly byte[] _jsonInt64Bytes = Encoding.UTF8.GetBytes(long.MaxValue.ToString());
+
+
+        private static readonly byte[] _jsonSingleBytes = Encoding.UTF8.GetBytes(float.MaxValue.ToString("F"));
+        private static readonly byte[] _jsonDoubleBytes = Encoding.UTF8.GetBytes(double.MaxValue.ToString("F"));
+        private static readonly byte[] _jsonDecimalBytes = Encoding.UTF8.GetBytes(decimal.MaxValue.ToString("F"));
+
+        private static readonly byte[] _jsonGuidBytes = Encoding.UTF8.GetBytes($"\"{Guid.Empty}\"");
+        private static readonly byte[] _jsonDateTimeBytes = Encoding.UTF8.GetBytes($"\"{DateTime.MaxValue:O}\"");
+        private static readonly byte[] _jsonDateTimeOffsetBytes = Encoding.UTF8.GetBytes($"\"{DateTimeOffset.MaxValue:O}\"");
+        private static readonly byte[] _jsonStringBytes = Encoding.UTF8.GetBytes($"\"{new string('a', 1024*2)}\"");
```

Comment:
I think this string is too long. I think a string like the "The quick brown fox jumps over the lazy dog" pangram would do here, and be closer in length to strings founds in typical payloads.

Reply:
Hmm, I think it might be useful to have both here. A longer string is going to give us protection against any bad scaling that might be introduced that we could miss with a shorter string.

Reply:
We could add
```cs
[Params(true, false)]
public bool UseLargeValue;
```
So we can test both scenarios, on `true` we use the `MaxValue` and on `false` we can use `default`.